### ISSUE TITLE
Using CSSGrid instead of bootstrap's

### ISSUE THREAD

**Issue by regisphilibert** [project-cx/coreui-hugo#3]
CSS grid is now supported by major browsers except IE11 and below.
This would allow us way more flexibility in our views where cards will take a big part
**From this:**
![capture d ecran 2018-01-23 10 30 35](https://user-images.githubusercontent.com/1480503/35284501-fbfcf6a8-0028-11e8-9bc6-d57a71f41d2b.png)
**to this:**
![capture d ecran 2018-01-23 10 33 12](https://user-images.githubusercontent.com/1480503/35284514-01ea304e-0029-11e8-9cad-845f02cd6f21.png)

**Bottom line: Do we need to support IE11 and below on this project?**

**Comment by DinisCruz** [comment]
no need to support any legacy browsers, in fact we can make a requirement to use the latest (Chrome) browser :)
